docs/source/conf.py

```diff
@@ -35,7 +35,7 @@
 extensions = [
     'sphinx.ext.autodoc',
     'sphinx.ext.intersphinx',
-    'sphinx.ext.mathjax',
+    'sphinx.ext.napoleon',
```

Comment:
@minrk Nice! This should allow nicer docstrings. Probably should uncomment and update Line 30 to '1.3'. Prior to '1.3' the extension was named differently.